packages/measured-reporting/README.md

```diff
@@ -15,20 +15,96 @@ npm install measured-reporting
+
+#### Current Implementations
+- [SignalFx Reporter](https://yaorg.github.io/node-measured/SignalFxMetricsReporter.html) in the `measured-signalfx-reporter` package.
+  - reports metrics to SignalFx.
+- [Logging Reporter](https://yaorg.github.io/node-measured/LoggingReporter.html) in the `measured-reporting` package.
+  - logs reported metrics at an info level through a supplied logger, defaulting to a new bunyan logger if none supplied.
+
+#### Creating an anonymous Implementation
 You can technically create an anonymous instance of this, see the following example.
 ```javascript
-// Create anonymous console logging Reporter instance.
-const reporter = new class extends Reporter {
+const os = require('os');
+const process = require('process');
+const { SelfReportingMetricsRegistry, Reporter } = require('measured-reporting');
+
+// Create a self reporting registry with a named anonymous Reporter instance;
+const registry = new SelfReportingMetricsRegistry(
+  new class ConsoleReporter extends Reporter {
+    constructor() {
+      super({
+        defaultDimensions: {
+          hostname: os.hostname(),
+          env: process.env['NODE_ENV'] ? process.env['NODE_ENV'] : 'unset'
+        }
+      })
+    }
+
     _reportMetrics(metrics) {
-        metrics.forEach(metric => console.log(JSON.stringify(metric.toJSON())))
+      metrics.forEach(metric => {
+        console.log(JSON.stringify({
+          metricName: metric.name,
+          dimensions: this._getDimensions(metric),
+          data: metric.metricImpl.toJSON()
+        }))
+      });
     }
-};
+  },
+);
+
+// create a gauge that reports the process uptime every second
+const processUptimeGauge = registry.getOrCreateGauge('node.process.uptime', () => process.uptime(), {}, 1);
+```
+
+Here is the output from that example
+
+```bash
+APP5HTD6ACCD8C:foo jfiel2$ NODE_ENV=development node index.js | bunyan
+[2018-06-06T23:39:49.678Z]  INFO: Reporter/9526 on APP5HTD6ACCD8C: {"metricName":"node.process.uptime","dimensions":{"hostname":"APP5HTD6ACCD8C","env":"development"},"data":0.092}
+[2018-06-06T23:39:50.685Z]  INFO: Reporter/9526 on APP5HTD6ACCD8C: {"metricName":"node.process.uptime","dimensions":{"hostname":"APP5HTD6ACCD8C","env":"development"},"data":1.099}
+[2018-06-06T23:39:51.690Z]  INFO: Reporter/9526 on APP5HTD6ACCD8C: {"metricName":"node.process.uptime","dimensions":{"hostname":"APP5HTD6ACCD8C","env":"development"},"data":2.104}
+[2018-06-06T23:39:52.691Z]  INFO: Reporter/9526 on APP5HTD6ACCD8C: {"metricName":"node.process.uptime","dimensions":{"hostname":"APP5HTD6ACCD8C","env":"development"},"data":3.105}
+[2018-06-06T23:39:53.692Z]  INFO: Reporter/9526 on APP5HTD6ACCD8C: {"metricName":"node.process.uptime","dimensions":{"hostname":"APP5HTD6ACCD8C","env":"development"},"data":4.106}
 ```
+
+
 It would of course be better to create a proper class and contribute it back as a new package for measured if it is generic and sharable.
```

Comment:
Maybe say "Consider creating a proper class and contributing it back to Measured if it is generic and sharable."